src/classes/ProfileResults.ts

```diff
@@ -142,35 +135,38 @@ module.exports = class ProfileResults {
         time_elapsed: 0,
         start_usage_bytes: 0,
         peak_usage_bytes: 0,
-        end_usage_bytes: 0
+        end_usage_bytes: 0,
       };
     }
 
-    const comparison = {
+    const comparison: Comparison = {
       time_elapsed: this.data.time_elapsed - obj.time_elapsed,
       start_usage_bytes: this.data.start_usage_bytes - obj.start_usage_bytes,
       peak_usage_bytes: this.data.peak_usage_bytes - obj.peak_usage_bytes,
-      end_usage_bytes: this.data.end_usage_bytes - obj.end_usage_bytes
+      end_usage_bytes: this.data.end_usage_bytes - obj.end_usage_bytes,
     };
 
     if (options.logResultsDiff) {
       Object.keys(comparison).forEach((k) => {
-        if (k === 'time_elapsed') return console.log(`Time elapsed: ${comparison[k] > 0 ? `+${comparison[k]}ms`.red : `${comparison[k]}ms`.green}`);
-        return console.log(`${(k[0].toUpperCase() + k.substr(1)).replace(/_/g, ' ')}: ${comparison[k] > 0 ? `+${bytes(comparison[k])}`.red : `-${bytes(-comparison[k])}`.green}`);
+        if (k === 'time_elapsed') return console.log(`Time elapsed: ${comparison[k] > 0 ? red(`+${comparison[k]}ms`) : green(`${comparison[k]}ms`)}`);
+        return console.log(`${(k[0].toUpperCase() + k.substr(1)).replace(/_/g, ' ')}: ${comparison[k] > 0 ? `+${red(bytes(Number(comparison[k])))}` : `-${green(bytes(-Number(comparison[k])))}`}`);
```

Comment:
These are some kinda crazy looking lines, they should be broken up and made more readable. The coloring functions could probably be split out and generalized. And we should take a few steps to build up the string and then log it at the end of the function.

Also, since `.log` returns `void` you should just have the return after since it obfuscates like this.